src/Entity/Workshop.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Entity;
+
+use DateTime;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity(repositoryClass="App\Repository\WorkshopRepository")
+ */
+class Workshop
+{
+    /**
+     * @ORM\Id
+     * @ORM\GeneratedValue
+     * @ORM\Column(type="integer")
+     *
+     * @var int
+     */
+    private $id;
+
+    /**
+     * @ORM\Column(type="string",length=200)
+     *
+     * @var string
+     */
+    private $title;
+
+    /**
+     * @ORM\Column(type="text")
+     *
+     * @var string
+     */
+    private $shortIntroduction;
+
+    /**
+     * @ORM\Column(type="text")
+     *
+     * @var string
+     */
+    private $description;
+
+    /**
+     * @ORM\Column(type="datetime")
+     *
+     * @var DateTime
+     */
+    private $createdAt;
+
+    public function __construct()
+    {
+        $this->createdAt = new DateTime();
+    }
+
+    public function getId(): int
+    {
+        return $this->id;
+    }
+
+    public function getTitle(): string
+    {
+        return $this->title;
+    }
+
+    public function setTitle(string $title)
+    {
+        $this->title = $title;
+    }
+
+    public function getShortIntroduction(): string
+    {
+        return $this->shortIntroduction;
+    }
+
+    public function setShortIntroduction(string $shortIntroduction)
+    {
+        $this->shortIntroduction = $shortIntroduction;
+    }
+
+    public function getDescription(): string
+    {
+        return $this->description;
+    }
+
+    public function setDescription(string $description)
+    {
+        $this->description = $description;
+    }
+
+    public function getCreatedAt(): DateTime
```

Comment:
We need setCreatedAt too @msvrtan

Reply:
Actually we dont want to allow setting it from the outside of the entity. It's value is set in __constructor() and should not be changed